src/inc/Request.php

```diff
@@ -212,13 +211,12 @@ public function getParameter($param, $default = '')
     public function getUrlElement($index, $default = '')
     {
         $index   = (int) $index;
-        $element = $default;
 
         if (isset($this->url_elements[$index])) {
-            $element = $this->url_elements[$index];
+            return $this->url_elements[$index];
```

Comment:
Same here. The "Happy Path" should be the one where we can get a URL-Element. So I'd switch that like for ```getParameter```

Reply:
Updated.